developer/js/tests/test-parse-wordlist.ts

```diff
@@ -69,17 +69,16 @@ describe('parseWordList', function () {
       ' hello ', //4, expect to trim whitespace
       'hello']; //5
 
-    const expected = [
-      [ 'hello', 10 /* 1+4+5 trimmed and identical */ ],
-      [ 'hell'+String.fromCharCode(0x00F3), 5 /* 2+3 normalised to NFC */ ]
-    ];
+    const expected: WordList = {};
+    expected['hello'] = 10; /* 1+4+5 trimmed and identical */
+    expected['hell'+String.fromCharCode(0x00F3)] = 5 /* 2+3 normalised to NFC */
```

Comment:
Why not use a literal here?

```suggestion
    const expected: WordList = {
        'hello': 10, /* 1+4+5 trimmed and identical */
        'hell\u00f3': 5, /* 2+3 normalised to NFC */
    };
```

Reply:
Great idea -- wanted to make it clear that it was an NFC form ... this is more concise.